project/src/main/java/com/google/sps/data/Candidate.java

```diff
@@ -14,4 +14,17 @@
 
 package com.google.sps.data;
 
-public class Candidate {}
+public class Candidate extends DirectoryCandidate{
```

Comment:
Nit: I'm not even 100% sure this is officially a rule but I think there should be a [space before the open bracket](https://google.github.io/styleguide/javaguide.html#s4.1-braces). It's never actually stated in the style guide, but is shown in all examples?